Calendar/js/app.js

```diff
@@ -0,0 +1,233 @@
+var _date = new Date();
+var _month = _date.getMonth();
+var _year = _date.getFullYear();
+var _calendar = document.getElementById('calendar');
+var _selectedMonthElement = document.getElementById('select-month');
+var _selectedYearElement = document.getElementById('select-year');
+var _weekElement = document.getElementsByClassName('week')[0];
+var _startYear;
+var _endYear;
+var _months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+
+var birthdays = {
+    "1" : {
+        "3" : ["Shubham Katariya"],
+        "15" : ["Darshana Shrivastava", "Shikha Shakarwar", "Vipin Sir"],
+        "31" : ["Sagar Mishra"]
+    },
+    "2" : {
+        "" : []
+    },
+    "3" : {
+        "" : []
+    },
+    "4" : {
+        "2" : ["Gurpreet Chhabra"],
+        "22" : ["Sonam Ravi Gupta"],
+        "30" : ["Ram Bhaiya"]
+    },
+    "5" : {
+        "3" : ["Siyaram Patidar"],
+        "8" : ["Shubham Choubey"],
+        "9" : ["Mayur Vaidya"],
+        "10" : ["Amit Nagar", "Deepak Patidar"],
+        "28" : ["Rahul Kulmi"]
+    },
+    "6" : {
+        "20" : ["Vishal Patidar"]
+    },
+    "7" : {
+        "6" : ["Awanish Tiwari"],
+        "21" : ["Surendra Patidar"],
+        "24" : ["Anjana Singh"]
+    },
+    "8" : {
+        "8" : ["Aaditya Paliwal"]
+    },
+    "9" : {
+        "" : [""]
+    },
+    "10" : {
+        "16" : ["Piyush Chandak"],
+        "19" : ["Rashmi Soni"]
+    },
+    "11" : {
+        "19" : ["Priyanshi Asawara"]
+    },
+    "12" : {
+        "3" : ["Abhijeet Bhowmik"],
+        "7" : ["Gourav Agrawal"],
+        "11" : ["Shashank Saxena"],
+        "12" : ["Nitesh Thakur", "Satya Narayan patidar"]
+    }
+}
+
+function createCalendar(_date, _month, _year) {
+    var startNumber = 1;
+    var lastDateOfPreviousMonth;
+    var totalRows;
+    var i = 1;
+    var j = 1;
+    var k = 1;
```

Comment:
Why declare `i`, `j` and `K`? Don't think you are using.

Reply:
Actually, I was using them to create calendar in single loop as anjana mam asked. I will remove them asap.